ehrapy/plot/ehrapy_plot/plot_qc.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from anndata import AnnData
+from rich.console import Console
+from rich.table import Table
+
+
+def qc_metrics(adata: AnnData, extra_columns: list[str] | None = None) -> None:
```

Comment:
Think we need to `# pragma: no cover` this function?

Reply:
Y, theres no proper way of testing this one.